torch/_inductor/kernel/flex_attention.py

```diff
@@ -733,20 +739,20 @@ def flex_attention(
 
 
 def flex_attention_backward_grid(
-    batch_size, num_heads, num_queries, d_model, num_key_value, meta
+    batch_size, q_heads, num_queries, d_model, kv_heads, num_key_value, meta
 ):
     """How is this kernel parallelized?
-    Currently this is only parallelizing over batch * num_heads, but we can, and want to
-    parallelize over ceil_div(num_key_value, key_value_block_size). To do this will either require
-    atomic updates to some grad values or to have a two pass kernel design.
+    Currently this is only parallelizing over batch* kv_heads, but we can, and want to
+    parallelize over ceil_div(q_heads//kv_heads * num_key_value, key_value_block_size).
```

Comment:
I think this comment is somewhat stale since we do now also slice up the num_queries and num keys